source/pervasive/vec.rs

```diff
@@ -160,6 +160,15 @@ impl<A> Vec<A> {
     {
         self.vec.clear();
     }
+
+    #[verifier(external_body)]
+    pub fn append(&mut self, other: &mut Vec<A>)
+        ensures
+            self@ == old(self)@ + other@,
```

Comment:
I think you mean this?
```suggestion
            self@ == old(self)@ + old(other)@,
```

Reply:
Oops, thank you!